sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -594,7 +594,7 @@ public async Task DownloadAsync_VersionId()
             Assert.IsNotNull(response.Value.Details.VersionId);
         }
 
-        // TODO: Recorded Only
+        [PlaybackOnly("Object Replication policies is only enabled on certain storage accounts")]
```

Comment:
Did the PlaybackOnly attribute work correctly?

Reply:
It actually doesn't. I just tested this now. Should I take it off then?

Reply:
Oh no!  Did it have any affect?

Reply:
no :confused:

Reply:
Ok let’s follow up on Monday, I’ll need to revisit my PR for the test attribute.